examples/java/src/main/java/glide/examples/StandaloneExample.java

```diff
@@ -0,0 +1,146 @@
+     *
+     * @return A <code>GlideClient</code> connected to the provided node address.
+     * @throws CancellationException if the operation is cancelled.
+     * @throws ExecutionException if the client fails due to execution errors.
+     * @throws InterruptedException if the operation is interrupted.
+     */
+    public static GlideClient createClient() throws CancellationException, ExecutionException, InterruptedException {
+        String host = "localhost";
+        Integer port = 6379;
+
+        // Check `GlideClientConfiguration` for additional options.
+        GlideClientConfiguration config =
+                GlideClientConfiguration.builder()
+                        .address(NodeAddress.builder().host(host).port(port).build())
+                        // Enable this field if the servers are configured with TLS.
+                        //.useTLS(true);
+                        .build();
+
+        GlideClient client = GlideClient.createClient(config).get();
+        return client;
+    }
+
+    /**
+     * Executes the main logic of the application, performing basic operations such as SET, GET, and
+     * PING using the provided <code>GlideClient</code>.
+     *
+     * @param client An instance of <code>GlideClient</code>.
+     * @throws ExecutionException if an execution error occurs during operations.
+     * @throws InterruptedException if the operation is interrupted.
+     */
+    public static void appLogic(GlideClient client) throws ExecutionException, InterruptedException {
+
+        // Send SET and GET
+        CompletableFuture<String> setResponse = client.set("foo", "bar");
+        Logger.log(Logger.Level.INFO, "app", "Set response is " + setResponse.get());
+
+        CompletableFuture<String> getResponse = client.get("foo");
+        Logger.log(Logger.Level.INFO, "app", "Get response is " + getResponse.get());
+
+        // Send PING to the primary node
+        CompletableFuture<String> pong = client.ping();
+        Logger.log(Logger.Level.INFO, "app", "Ping response is " + pong.get());
+    }
+
+    /**
+     * Executes the application logic with exception handling.
+     *
+     * @throws ExecutionException if an execution error occurs during operations.
+     */
+    private static void execAppLogic() throws ExecutionException {
+
+        while (true) {
+            GlideClient client = null;
+            try {
+                client = createClient();
+                appLogic(client);
+
+            } catch (CancellationException e) {
+                Logger.log(Logger.Level.ERROR, "glide", "Request cancelled: " + e.getMessage());
+                throw e;
+            } catch (InterruptedException e) {
+                Logger.log(Logger.Level.ERROR, "glide", "Client interrupted: " + e.getMessage());
+                Thread.currentThread().interrupt(); // Restore interrupt status
+                throw new CancellationException("Client was interrupted.");
```

Comment:
why do we need to catch InterruptedException and change it to CancellationException?

Reply:
As part of Java async API, we need to handle InterruptedException. After discussing with team,  `CancellationException` might be the closest handling I can use. Also, in this case, I cannot use `throw e` like other handling.